### PR TITLE
Fix bugs in session-4

### PR DESCRIPTION
**Pull request by off99555**
Providing hotfix

### CODE REVIEW

File: session-4/lecture-4.ipynb [hunk 1]
```diff
@@ -2056,7 +2056,7 @@
     }
    ],
    "source": [
-    "features = [name for name in names if 'maxpool' in name.split()[-1]]\n",
+    "features = [name for name in names if 'maxpool' in name.split('/')[-1]]\n",
```
Comment (off99555) [inline session-4/lecture-4.ipynb]: The `.split('/')` is not necessary but I think you were trying to split the layer name by `/` but you forget to put it.

File: session-4/lecture-4.ipynb [hunk 2]
```diff
@@ -4581,8 +4581,8 @@
     "with tf.Session(graph=g) as sess, g.device('/cpu:0'):\n",
     "    res = softmax.eval(feed_dict={\n",
     "        x: img_4d,\n",
-    "        'vgg/dropout_1/random_uniform:0': [[1.0]],\n",
-    "        'vgg/dropout/random_uniform:0': [[1.0]]})[0]\n",
+    "        'vgg/dropout_1/random_uniform:0': [[1.0] * 4096],\n",
+    "        'vgg/dropout/random_uniform:0': [[1.0] * 4096]})[0]\n",
```
Comment (off99555) [inline session-4/lecture-4.ipynb]: If I do not do this then the error happens:
`ValueError: Cannot feed value of shape (1, 1) for Tensor 'vgg/dropout_1/random_uniform:0', which has shape '(?, 4096)'`

File: session-4/lecture-4.ipynb [hunk 3]
```diff
@@ -3884,7 +3884,7 @@
     "# Let's get a gaussian filter\n",
     "from scipy.ndimage.filters import gaussian_filter\n",
     "\n",
-    "# Which we'll smooth with a standard deviation of 0.5\n",
+    "# Which we'll smooth with a standard deviation of 1.0\n",
```
Comment (off99555) [inline session-4/lecture-4.ipynb]: Make it the same as the code

**Comment by pkmital** [comment]
TY!

**Pull request merged**